suse_migration_services/migration_config.py

```diff
@@ -103,6 +103,9 @@ def is_debug_requested(self):
     def is_zypper_migration_plugin_requested(self):
         return self.config_data.get('use_zypper_migration', True)
 
+    def is_reboot_requested(self):
```

Comment:
To me this reads as if it implies that a system can be migrated without a reboot. "kexec" is a form of a reboot. I think a different name may help clarify: "use_reboot_exec" may be one option to more explicitly express the intend.